PLEP-0007.rst

```diff
@@ -0,0 +1,262 @@
+|   namespace.  For example, linear, langmuir, magnetic-flux, etc. analysis    |
+|   tools should be placed in their respective namespaces.                     |
++------------------------------------------------------------------------------+
+| .. _diagnostics:                                                             |
+|                                                                              |
+| **diagnostics**                                                              |
++------------------------------------------------------------------------------+
+| :directory: ``./plasmapy/diagnostics/``                                      |
+| :access: ``plasmapy.diagnostics``                                            |
+| :PLEP:                                                                       |
+| :scope:                                                                      |
+|   A sub-package that fully defines the parameters of a "diagnostic."  A      |
+|   "diagnostic" is any data collecting instrument ranging from experimental   |
+|   probes, like Langmuir and magnetic flux probes, to analogous synthetic     |
+|   diagnostics for simulations.                                               |
+|                                                                              |
+|   Tools in this package do not define any analysis routines, but focus on    |
+|   defining probe characteristics/calibrations that can be seamlessly passed  |
+|   to ``plasmapy.analysis`` routines.                                         |
+|                                                                              |
+|   A "``Diagnostic``" class should fully define its diagnostic                |
+|   characteristics and provide access to its most relevant analysis tools     |
+|   found in ``plasmapy.analysis``.                                            |
++------------------------------------------------------------------------------+
+| .. _dispersion:                                                              |
+|                                                                              |
+| **dispersion**                                                               |
++------------------------------------------------------------------------------+
+| :directory: ``./plasmapy/dispersion/``                                       |
+| :access: ``plasmapy.dispersion``                                             |
+| :PLEP:                                                                       |
+| :scope:                                                                      |
+|   A sub-package dedicated to solving various plasma dispersion relations.    |
+|                                                                              |
+|   All dispersion solvers defined in this sub-package should be accessible    |
+|   through a dispersion solver factory class, ``DispersionSolver``.           |
```

Comment:
I'm still inclined to not include implementation details in this PLEP, since if we think of a better way then we wouldn't have to go through the process of changing this PLEP.

Reply:
I get that.  I'll go ahead and remove this statement.